### ISSUE TITLE
Support for RECE4244(RMX2195)

### ISSUE THREAD

**Issue by IYSTREEM** [Neternels/devices#6]
i have been looking for a nethunterkernel for my phone for few days i could not find any can you please make 1 for it?

**Comment by cyberknight777** [comment]
No. Request it in https://t.me/neternels_chat or build it yourself.
